crates/trie/trie/src/walker.rs

```diff
@@ -148,8 +148,8 @@ impl<C: TrieCursor> TrieWalker<C> {
     ///
     /// # Returns
     ///
-    /// * `Result<Option<Nibbles>, Error>` - The next key in the trie or an error.
-    pub fn advance(&mut self) -> Result<Option<Nibbles>, DatabaseError> {
+    /// * `Result<(), Error>` - Unit on success or an error.
+    pub fn advance(&mut self) -> Result<(), DatabaseError> {
```

Comment:
can we add a note here that the next key in the trie can then be obtained via `key()`?

Reply:
Added 🙏.